src/initializeApp.js

```diff
@@ -18,8 +18,11 @@ function setOptions(urlParams, store) {
   if (urlParams.rpc) {
     store.commit(`setRpcUrl`, urlParams.rpc)
   }
-  if (config.mobileApp || urlParams.insecure) {
-    store.commit(`setInsecureMode`)
+  if (config.mobileApp || urlParams.insecure === `true`) {
+    store.commit(`setInsecureMode`, true)
+  }
+  if (urlParams.insecure === `false`) {
```

Comment:
This is the default, why set it?

Reply:
The flag can be set to true in store and you should also have the possibility to change to false via url param. I'm happy to revert it if you find it's not useful.